### PR TITLE
Fix install on Mac.

### PR DESCRIPTION
**Pull request by sean-mcmanus**
Fix install on Mac.

### CODE REVIEW

File: Extension/src/main.ts [hunk 1]
```diff
@@ -306,7 +306,7 @@ function makeOfflineBinariesExecutable(installBlob: InstallBlob): Thenable<void>
     packages.forEach(p => {
         if (p.binaries && p.binaries.length > 0 &&
             p.platforms.findIndex(plat => plat === installBlob.info.platform) !== -1 &&
-            p.architectures.findIndex(arch => arch === installBlob.info.architecture) !== - 1) {
+            (p.architectures == undefined || p.architectures.findIndex(arch => arch === installBlob.info.architecture) !== - 1)) {
```
Comment (bobbrow) [inline Extension/src/main.ts]: always use `===` over `==`.  Is there a linting rule for this?

**Review by bobbrow — CHANGES_REQUESTED** [review]

File: Extension/src/main.ts (hunk 1 above)
Comment (WardenGnaw) [reply Extension/src/main.ts]: Yep. ```"triple-equals": true```

Comment (sean-mcmanus) [reply Extension/src/main.ts]: Ok.

Comment (sean-mcmanus) [reply Extension/src/main.ts]: So should I add triple-equal to tslint and fix all the non-triple equals (a lot)?

Comment (sean-mcmanus) [reply Extension/src/main.ts]: I'll do that....

Comment (WardenGnaw) [reply Extension/src/main.ts]: Talked with @sean-mcmanus, delaying the triple-equal tslint to another PR.

Comment (sean-mcmanus) [reply Extension/src/main.ts]: I just changed this 1 instance to ===.

**Review by WardenGnaw — APPROVED** [review]

**Review by bobbrow — APPROVED** [review]

**Pull request merged**